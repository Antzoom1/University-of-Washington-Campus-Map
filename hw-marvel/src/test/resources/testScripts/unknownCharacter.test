# Finds a path with one character in staffSuperHeroes dataset and one
# unknown character

LoadGraph staffSuperheroes staffSuperheroes.csv

FindPath staffSuperheroes Perkins-the-Magical-Singing-Instructor OnePunchMan