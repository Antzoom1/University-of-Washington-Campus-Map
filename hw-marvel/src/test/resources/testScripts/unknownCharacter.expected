# Finds a path with one character in staffSuperHeroes dataset and one
# unknown character

loaded graph staffSuperheroes

unknown: OnePunchMan

